src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -241,6 +242,9 @@ private function getCheckOptions(InputInterface $input): Options
         if (is_string($fileName) === false) {
             return new Options();
         }
+        if (file_exists($fileName) === false) {
+            return new Options();
+        }
```

Comment:
I think it's best for the tool to just crash, when the given file is not found.

This shows an interesting case to be tested:

 * file does not exist
 * default setting says to look in `composer-require-checker.json`

I think this highlights that perhaps `composer-require-checker.json` as a default command value is not sufficient.

A test with `--config-file=not-existing.json` that **throws** should be added here.

Reply:
Something like `if (! file_exists(...) && $file === self::DEFAULT_FILE) {` would suffice (with the testing endorsed above)

Reply:
Done